third_party/cpaembedded/embedded/kiro_converter.go

```
@@ -0,0 +1,492 @@
+
+// parseKiroRequest parses an Anthropic Messages JSON request body.
+func parseKiroRequest(raw []byte) (kiroRequest, error) {
+	var request kiroRequest
+	if err := json.Unmarshal(raw, &request); err != nil {
+		return kiroRequest{}, fmt.Errorf("parse Kiro request: %w", err)
+	}
+	return request, nil
+}
+
+// normalizeKiroSystem flattens the Anthropic system (string or list) to a single string.
+func normalizeKiroSystem(system interface{}) string {
```

golangci-lint (2.13.2)

[error] func normalizeKiroSystem is unused

(unused)